src/mixins/_button.scss

```diff
@@ -89,6 +91,62 @@
     transform: none;
   }
 
+  .c-button__extra {
+    display: flex;
+    font-size: ms.step(1);
+    margin: 0 ms.step(-4);
+    position: relative;
+
+    &:first-child {
+      margin-left: ms.step(-4) * -1;
+    }
+
+    &:last-child {
+      margin-right: ms.step(-4) * -1;
+    }
+  }
+
+  /**
+   * Toggling button
+   * This adds a slash across the icon to visually show the state has changed
+   * when `aria-pressed` is `'true'`.
+   */
+  &[aria-pressed='true'] {
+    .c-button__extra {
+      &::after {
+        background: currentColor;
+        border-radius: size.$border-radius-full;
+        content: '';
+        height: size.$edge-medium;
+        left: 0;
+        position: absolute;
+        top: 50%;
+        transform: rotate(-30deg);
+        width: 100%;
+      }
+
+      $clip-path: polygon(
+        -5% -5%,
+        105% -5%,
+        105% 23%,
+        -5% 87%,
+        -5% 101.4711%,
+        105% 37.9626%,
+        105% 105%,
+        -5% 105%
+      );
+
+      & > * {
+        clip-path: $clip-path;
+        transform: translate(0, 0);
```

Comment:
Can we add a comment mentioning that this is to fix a bug where some browsers would render the element as slightly offset from others?

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1298/commits/44bf076b5a171a9c727f021becf49d5f12f32a18